server.go

```diff
@@ -80,6 +80,10 @@ type server struct {
 	quit chan struct{}
 }
 
+type SendMessages func(target *btcec.PublicKey, msg ...lnwire.Message) error
```

Comment:
I don't think these need to be type-def'd as new functions. At the site where these are actually used (the `fundingManager`) the use of a new type makes the inner workings of the function a bit more opaque, the reader needs to jump files in order to see exactly what a `SendMessages` type is.